mend_sca_cleanup_tool/sca_cleanup_tool.py

```diff
@@ -382,31 +387,46 @@ def parse_config_file(filepath):
                     project_parallelism_level=config['DEFAULT'].get('ProjectParallelismLevel', 5),
                     dry_run=config['DEFAULT'].getboolean("DryRun", False),
                     skip_report_generation=config['DEFAULT'].getboolean("SkipReportGeneration", False),
-                    skip_project_deletion=config['DEFAULT'].getboolean("SkipProjectDeletion", False)
+                    skip_project_deletion=config['DEFAULT'].getboolean("SkipProjectDeletion", False),
+                    proxy=get_config_file_value(config['DEFAULT'].get("ProxyUrl"),"")
                 )
     else:
         print(f"No configuration file found at: {filepath}")
         exit()
 
 
 def call_api(data, header=None, method="POST", report=False):
+    global WARNING_MSG
     if header is None:
         header = HEADERS
     try:
-        res_request = requests.request(
-            method=method,
-            url=f"https://{CONFIG.mend_url}{API_VER}",
-            data=data,
-            headers=header,
-            )
-        res = res_request.content if report else res_request.text
+        proxy = {"https": CONFIG.proxy, "http": CONFIG.proxy} if CONFIG.proxy else {}
+        with warnings.catch_warnings(record=True) as warning_list:
+            warnings.simplefilter("always", InsecureRequestWarning)
+            res_request = requests.request(
+                method=method,
+                url=f"https://{CONFIG.mend_url}{API_VER}",
+                data=data,
+                headers=header,
+                proxies=proxy,
+                verify=False
+                )
+            res = res_request.content if report else res_request.text
     except Exception as err:
         sys.exit(f'Exception was raised: {err}')
+
+    if not WARNING_MSG:
+        for warning in warning_list:
+            if issubclass(warning.category, InsecureRequestWarning):
+                index_of_see = str(warning.message).find("See:")
+                print(f"Warning: {str(warning.message)[:index_of_see].strip()}")
+                WARNING_MSG = True
+
     return res
 
 
 def remove_invalid_chars(string_to_clean):
-    return re.sub('[:*<>/"?|.]', '-', string_to_clean).replace("\\", "-")
+    return re.sub('[:*<>/"?|]', '-', string_to_clean).replace("\\", "-")  # The "." is valid char
```

Comment:
The comment is unneeded here since the . is not included in the regex. The presence of the comment will be confusing in the future.